CHANGELOG.md

```diff
@@ -8,7 +8,15 @@ The romcal v3 is a major rewrite of the core library, with performance and archi
 As a result, romcal v3 is now up to 10x faster than v2 and v1.
 The new architecture helped to introduce new features without performance and code quality impacts.
 
-### Added
+### v 3.0.0@dev-52
+
+#### Breaking change
+
+- The `OPTIONAL_MEMORIAL` `rank` is being reintroduced, which was previously available as `OPT_MEMORIAL` until Romcal v2. This re-introduction includes related localizations for all currently supported locales.
```

Comment:
I probably use `rank` here as a term not as a code. In case you still want to use it as a code, IMHO it should be written as `Rank`.

```suggestion
- The `OPTIONAL_MEMORIAL` rank is being reintroduced, which was previously available as `OPT_MEMORIAL` until Romcal v2. This re-introduction includes related localizations for all currently supported locales.
```

Reply:
`rank` was meant to refer to the `rank` property, not the `Rank` type. But alright, I'm going with your suggestion 👍👍. Anyway, you have a great eye for detail (which is excellent), but let's merge this PR now and continue to make progress on other matters.